libraries/common/message-cache.js

```diff
@@ -177,7 +177,7 @@ export async function openMessageCache(cookieStoreId, forceClear) {
  */
 export async function updateMessages(cookieStoreId, forceClear, username, xToken) {
   await openMessageCache(cookieStoreId, forceClear);
-  if (username === null) return [];
+  if (typeof username === "undefined" || username === null) return [];
```

Comment:
Why do we need to change this file? Is it actually related to the other changes?

Reply:
yeah, now it will pass in undefined vs passing in null. it matters